browser/src/control/Control.StatusBar.js

```diff
@@ -247,6 +247,7 @@ class StatusBar extends JSDialog.Toolbar {
 			{type: 'toolitem', id: 'signstatus', command: '.uno:Signature', w2icon: '', text: _UNO('.uno:Signature'), visible: false},
 			{type: 'spacer',  id: 'permissionspacer'},
 			this._generateHtmlItem('documentstatus', 2),					// spreadsheet, text, presentation, drawing
+			{type: 'customtoolitem',  id: 'multi-page-view', command: 'multipageview', text: _('Multi Page View'), dataPriority: 10,  visible: false}, // text
```

Comment:
somehow it is hidden in my case and I cannot get it back, possibly due to `dataPriority:` ?
I see `status-hidden` which is done by https://github.com/CollaboraOnline/online/blob/96804eb4cd492470e0c6ae8f5f65850b355235b0/browser/src/control/jsdialog/Util.ScrollableBar.ts#L129

yes, I did remove the patch hiding it explicitly :)